docs/src/examples/components/Portal/Types/PortalExampleFocusTrapped.tsx

```diff
@@ -25,6 +25,10 @@ class PortalExamplePortal extends React.Component {
             // When 'false', all clicks outside the Portal will be caught and not handled.
             // 'true' by default.
             isClickableOutsideFocusTrap: false,
+            // Indicates whether to focus element which triggered FTZ on outside click
+            // 'false' by default
+            // note: if isClickableOutsideFocusTrap === 'false', focusTriggerOnOutsideClick will not be taken into account.
+            focusTriggerOnOutsideClick: false,
```

Comment:
should be removed, as it doesn't take any effect for this example

Reply:
this example just shows possible API that can be used for FocusTrapZone as there is no separate component as ```FocusTrapZone``` that clients can lookup for props, unless they'll check code directly.
And it also gives a possibility to play with it, by setting different values for props.
Let me know what you think @miroslavstastny @kuzhelov

Reply:
I'll create an issue for creating proper documentation with examples for FoucsTrapZone and FocusZone. But would leave this change as it is.